src/maintenance/packages/mirrors.md

```diff
@@ -0,0 +1,48 @@
+# Mirrors
+
+## Available mirrors
+
+### Tier 1 Mirrors
+
+| Repository | Location |
+|:--------|:-------:|
+| [http://alpha.de.repo.voidlinux.org](http://alpha.de.repo.voidlinux.org) | EU: Germany |
+| [http://beta.de.repo.voidlinux.org](http://beta.de.repo.voidlinux.org) | EU: Germany |
+| [http://alpha.us.repo.voidlinux.org](http://alpha.us.repo.voidlinux.org) | USA: Kansas City |
+| [http://mirror.clarkson.edu/voidlinux/](http://mirror.clarkson.edu/voidlinux/) | USA: New York |
+| [http://mirrors.servercentral.com/voidlinux/](http://mirror.clarkson.edu/voidlinux/) | USA: Chicago |
+
+### Tier 2 Mirrors
+
+
+| Repository | Location |
+|:--------|:-------:|
+| [http://mirror.aarnet.edu.au/pub/voidlinux/](http://mirror.aarnet.edu.au/pub/voidlinux/) | AU: Canberra |
+| [http://ftp.swin.edu.au/voidlinux/](http://ftp.swin.edu.au/voidlinux/) | AU: Melbourne |
+| [http://ftp.acc.umu.se/mirror/voidlinux.eu/](http://ftp.acc.umu.se/mirror/voidlinux.eu/) | EU: Sweden |
+| [https://mirrors.dotsrc.org/voidlinux/](https://mirrors.dotsrc.org/voidlinux/) | EU: Denmark |
+| [http://www.gtlib.gatech.edu/pub/VoidLinux/](http://www.gtlib.gatech.edu/pub/VoidLinux/) | USA: Atlanta |
+
+
+
+## Changing mirrors
+
+Copy all repository configuration files from `/usr/share/xbps.d` to `/etc/xbps.d` and then change the repository urls in `/etc/xbps.d`.
+
+```
+# mkdir -p /etc/xbps.d
+# cp /usr/share/xbps.d/*-repository-*.conf /etc/xbps.d/
+# sed -i 's|https://repo.voidlinux.eu/current|<repository>|g' /etc/xbps.d/*-repository-*.conf
```

Comment:
It would be nice to provide a command that works out of the box, but this is liable to become out of date. Perhaps a simple recommendation to edit the copied files directly would be best here.

Reply:
The long term goal is to default to an automatic redirection mirror service.  This service is actually live right now (auto.voidlinux.org) but isn't stable enough to recommend yet.

It would not be too difficult to provide a mirror configuration and chooser program if anyone wanted to write one.